.env.template

```diff
@@ -4,4 +4,7 @@ ENV=testing
 GUILD_ID=1234567890
 MOD_CHANNEL_ID=0987654321
 DEV_CHANNEL_ID=0987654321
+CAREER_QUESTIONS_CHANNEL_ID=0987654321
+RESUME_REVIEW_CHANNEL_ID=0987654321
+SHARE_YOUR_CONTENT_CHANNEL_ID=0987654321
```

Comment:
As discussed on Discord, I believe the auto-thread feature should also be applied to `#share-your-content` and `#hiring`

Reply:
That's a lot more specific. Renamed the PR to that.

Will add both channels to the code in the next revision.

Reply:
Because it's fairly dynamic I think it would be better to have a database entry of like a 1 col table where you can add/remove channels by command. Never know if we plan to add or remove channels from this practice